torch/onnx/__init__.py

```diff
@@ -149,20 +149,73 @@ def _newNode(self, opname, *args, **kwargs):
     return n
 
 
-def _op(self, opname, *args, **kwargs):
+def _op(self, opname, *raw_args, **kwargs):
+    outputs = kwargs.pop('outputs', 1)
+    def const_if_tensor(arg):
+        if isinstance(arg, torch._C.Node):
+            return arg
+        else:
+            return self.op("Constant", value_z=arg)
+    args = list(const_if_tensor(arg) for arg in raw_args)
+    n = self.appendNode(_newNode(self, opname, *args, **kwargs))
+    if outputs == 1:
+        return n
+    return tuple(self.appendNode(self.createSelect(n, i)) for i in _range(outputs))
+
+
+# Note [Export inplace]
+# ~~~~~~~~~~~~~~~~~~~~~
+# In abstract, it would be better for us to export inplace annotations,
+# than to not export them, since it is useful information that can
+# help the target of an ONNX export export more efficiently.  However,
+# ONNX doesn't currently formalize inplace.  Fortunately, it's sound to drop
+# inplace annotations, but we are losing information this way.
+
+
+_current_graph = None
+
+
+def run_symbolic_function(g, n, inputs):
+    import torch.onnx.symbolic
+    global _current_graph
+    assert not _current_graph
+    _current_graph = g
+    # TODO: Monkeypatch this on Node or implement it directly in the bindings
+    def get_node_attr(n, k):
```

Comment:
I think @apaszke  and I converged on enabling `foo[attr]` to do exactly this (`__getitem__` only). So monkey patch it if you desire.